src/Sentry/Scope.cs

```diff
@@ -382,6 +384,19 @@ public void ClearBreadcrumbs()
 #endif
     }
 
+    internal void ApplyToSpan(ISpanData spanData)
+    {
+        if (Request.Method is {} method)
+        {
+            spanData.SetExtra(OtelSemanticConventions.AttributeHttpRequestMethod, method);
+        }
+
+        if (Contexts.Response.StatusCode is {} statusCode)
+        {
+            spanData.SetExtra(OtelSemanticConventions.AttributeHttpResponseStatusCode, statusCode);
```

Comment:
This will currently it will overwrite these values, if they've already been set (e.g. by a `SentryMessageHandler`). Need to add a TryAdd method to IHasExtra and use this instead.